test/regex/lib/util.js

```diff
@@ -8,7 +8,9 @@ const { RegExpParser } = require('regexpp');
  * @typedef {{ pattern: Pattern, flags: Flags }} LiteralAST
  */
 
-const parser = new RegExpParser({ strict: false, ecmaVersion: 6 });
+const parser = new RegExpParser({ strict: false, ecmaVersion: 2018 });
```

Comment:
`ecmaVersion` was always expeccted to be a year here, so this should at least be 2015. I updated this to 2018 (ES 9) for unicode property escape support